src/main/java/io/cdap/plugin/servicenow/sink/model/APIResponse.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.servicenow.sink.model;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Model class for API Response from ServiceNow Table APIs
+ */
+public class APIResponse {
+  
+  private List<Map<String, String>> result;
+
+  public List<Map<String, String>> getResult() {
+    return result;
+  }
+
+  public void setResult(List<Map<String, String>> result) {
```

Comment:
In general, avoid using setters, especially when the class is supposed to represent something immutable. Otherwise it is easy to introduce bugs, where one class reads the value, another class sets it to something else, and now it is unclear what the true state should be.

Instead, variables should all be final and set in a constructor.

Reply:
Added constructor. Removed the setter methods. cbfb212c77242fe61b2ef14c690a9079b6a9e22f

Reply:
remove this setter

Reply:
Removed. 0ff7d2955ec7a6844c0ee368dec07e59c388c2c8